Tasks/JenkinsQueueJob/task.json

```diff
@@ -80,7 +80,16 @@
             "resizable": "true",
             "rows" : "4"
         }
-      }
+      },
+      {
+        "name": "trustSSL",
+        "type": "boolean",
+        "label": "Trust server certificate",
+        "defaultValue": "false",
+        "required": true,
```

Comment:
required can be false here, you can default to not trusting the cert (which you already do)